api/wasm/cpp/proxy_wasm_externs.h

```diff
@@ -16,6 +16,7 @@
    extern "C" EMSCRIPTEN_KEEPALIVE void proxy_onStart(uint32_t root_context_id, uint32_t root_id_ptr, uint32_t root_id_size);
    extern "C" EMSCRIPTEN_KEEPALIVE void proxy_onConfigure(uint32_t root_context_id, uint32_t configuration_ptr, uint32_t configuration_size);
    extern "C" EMSCRIPTEN_KEEPALIVE void proxy_onTick(uint32_t root_context_id);
+   extern "C" EMSCRIPTEN_KEEPALIVE void proxy_onQueueReady(uint32_t root_context_id, uint32_t queue_name_ptr, uint32_t queue_name_size);
```

Comment:
Nit: `proxy_onSharedQueueReady`.

Reply:
Also, could we perhaps get `queue_id` upon registration and have `proxy_onSharedQueueReady` use it instead of the string?

Reply:
That seems reasonable.  Do you want an "getSharedQueue(vm_id, queue_name)" which would return a queue_id as well for the sender?

Reply:
Would we use that before each call or once during startup? i.e. could the `queue_id` change with new registration?

Reply:
Just during setup.